nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -1553,6 +1556,9 @@ def __parse_item_rttm(self, line: str, manifest_file: str) -> Dict[str, Any]:
         else:
             item['audio_file'] = os.path.expanduser(item['audio_file'])
 
+        if 'rttm_filepath' not in item:
```

Comment:
Let's use `get_full_path` for all file paths so that relative paths can be properly handled: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/parts/preprocessing/manifest.py#L165